src/streams/readStream.ts

```diff
@@ -83,7 +83,7 @@ Client.prototype.readStream = async function (
 
   options.setStream(streamOptions);
   options.setResolveLinks(resolveLinkTos);
-  options.setCount(count.toString(10));
+  options.setCount((maxCount ?? Number.MAX_SAFE_INTEGER).toString(10));
```

Comment:
Considering this is a `uint64`, does `9007199254740991` match the default of other clients? 
Prehaps a `BigInt` of the same count should be used?

Reply:
I think that `Number.MAX_SAFE_INTEGER` is more self-explanatory. It's just an arbitrary value for unbound `maxCount`. I don't think that EventStoreDB will be able to return more than `Number.MAX_SAFE_INTEGER`. If we use e.g. `9007199254740991` then we would need to explain to the users what does that value mean.